src/BusinessLogic/DataAccess/GeneralSettings/Entities/GeneralSettings.php

```diff
@@ -44,7 +44,11 @@ public function inflate(array $data): void
             (bool)self::getArrayValue($generalSettings, 'showSeQuraCheckoutAsHostedPage', null),
             static::getDataValue($generalSettings, 'allowedIPAddresses', []),
             static::getDataValue($generalSettings, 'excludedProducts', []),
-            static::getDataValue($generalSettings, 'excludedCategories', [])
+            static::getDataValue($generalSettings, 'excludedCategories', []),
+            (bool) self::getArrayValue($generalSettings, 'enabledForServices', false),
+            (bool) self::getArrayValue($generalSettings, 'allowFirstServicePaymentDelay', false),
+            (bool) self::getArrayValue($generalSettings, 'allowServiceRegistrationItems', false),
+            (string) self::getArrayValue($generalSettings, 'defaultServiceEndDate', null)
```

Comment:
```suggestion
            (string) self::getArrayValue($generalSettings, 'defaultServicesEndDate', null)
```
Missing s defaultService`s`EndDate ?